tlsn/tlsn-common/src/config.rs

```diff
@@ -0,0 +1,44 @@
+//! TLSNotary protocol config and config utilities.
+
+use crate::Role;
+
+/// Default for the maximum number of bytes that can be sent (4Kb).
+pub const DEFAULT_MAX_SENT_LIMIT: usize = 1 << 12;
+/// Default for the maximum number of bytes that can be received (16Kb).
+pub const DEFAULT_MAX_RECV_LIMIT: usize = 1 << 14;
```

Comment:
What is the difference between these and `DEFAULT_TRANSCRIPT_MAX_SIZE` in `tls-mpc/src/config.rs`?